### PR TITLE
Feature position options

### PR DESCRIPTION
**Pull request by SamHoulder**
Widget digitalposition
Added displayoption DD M.mmmm Dir 
Added to default paths: navigation.courseRhumbline.nextPoint, navigation.courseRhumbline.lastPoint

### CODE REVIEW

File: lib/streambundle.js [hunk 1]
```diff
@@ -56,6 +56,7 @@ StreamBundle.prototype.handleDelta = function(delta, instrumentPanel) {
   var that = this;
   delta.updates && delta.updates.forEach(function(update) {
     var sourceId = getSourceId(update.source);
+    // if (sourceId === 'no_source') return;
```
Comment (cmotelet) [inline lib/streambundle.js:59]: Please remove this unused line

File: lib/ui/widgets/digitalposition.js [hunk 2]
```diff
@@ -97,6 +117,14 @@ var positionFormats = [
       label: "DDirM'S''ddd",
       fontSize: 15
     }
+  },
+  function(positionObject) {
+    return {
+      latitude: displayDDM(positionObject.latitude, coordTypeLatitude, false),
+      longitude: displayDDM(positionObject.longitude, coordTypeLongitude, false),
+      label: "Dir D\u00b0 M.mmm'",
```
Comment (cmotelet) [inline lib/ui/widgets/digitalposition.js:125]: As you display the position in this format `59° 43.2601' N` change the label to: `D\u00b0 M.dddd' Dir`
You can also display like that to save one chr `59° 43'2601 N` and label to: `D\u00b0 M'dddd Dir`

**Review by cmotelet — CHANGES_REQUESTED** [review]

File: lib/streambundle.js (hunk 1 above)
Comment (SamHoulder) [reply lib/streambundle.js]: Removed in last commit.

File: lib/ui/widgets/digitalposition.js (hunk 2 above)
Comment (SamHoulder) [reply lib/ui/widgets/digitalposition.js]: My goal here was to match OPNs "Degrees, decimal minutes", as this format is commonly used here. 
I've corrected the label to D\u00b0 M.mmmm' Dir

**Pull request merged**